src/sentry/web/frontend/react_page.py

```diff
@@ -69,6 +69,17 @@ class ReactMixin:
     def meta_tags(self, request: Request, **kwargs):
         return {}
 
+    def preconnect(self) -> list[str]:
+        preconnects = []
+        if (
+            # In dev (and possibly other configs), STATIC_HOST is not set, not checking for it would result in
+            # preconnects to the server we are already receiving the response from.
+            getattr(settings, "STATIC_ORIGIN", None)
+            is not None
+        ):
+            preconnects.append(settings.STATIC_ORIGIN)
```

Comment:
You should add `STATIC_ORGIN` to `server.py` with a default value of `None`. That would let you skip the `getattr` call and let you document the setting better.

Reply:
Ah, thank you! I was wondering why it didn't scream when I tried accessing a different value. I'll update it, thank you!